.vib/kube-prometheus/cypress/cypress/integration/kube_prometheus_spec.js

```diff
@@ -20,7 +20,7 @@ it('allows executing a query and displaying response data for each service monit
   Object.keys(monitors).forEach((jobName, i) => {
     const query = Object.values(monitors)[i].query;
 
-    cy.get('[role="textbox"]').clear().type(query);
+    cy.get('[role="textbox"]').clear().type(query+'{enter}');
```

Comment:
Prefer [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for interpolation over concatenation. 

```suggestion
    cy.get('[role="textbox"]').clear().type(`${query}{enter}`);
```

Reply:
As mentioned, include the `{force: true}` option in the `clear` command to increase resiliency.

Reply:
Also, replicate the changes in the test above.